components/scream/scripts/test_all_scream.py

```diff
@@ -154,23 +154,23 @@ def __init__(self, cxx, kokkos=None, submit=False, parallel=False, fast_fail=Fal
         #   Deduce how many testing resources per test   #
         ##################################################
 
-        if ctest_parallel_level > 0:
-            ctest_max_jobs = ctest_parallel_level
+        if int(ctest_parallel_level) > 0:
```

Comment:
Weren't these already ints?

Reply:
I thought so, but when I ran it with a custom value `--make-parallel-level 128`, python said comparison between str and int wasn't allowed. The default was 0, so I thought the type was setlled to int. But googling a bit revealed someone had a similar problem, and they told him to cast. Maybe the input parser stores everything as a string?

Reply:
Ah, then we need to tell the argparser that these options should be treated as ints.

Reply:
Here's an example of int option:
```
    parser.add_argument("-i", "--int-opt", type=int, action="store",
                      dest="int_opt", default=5,
                      help="Specifies an int option.")
```